ports/atmel-samd/boards/circuitbrains_basic_m0/mpconfigboard.h

```diff
@@ -0,0 +1,41 @@
+#define MICROPY_HW_BOARD_NAME "CircuitBrains Basic"
+#define MICROPY_HW_MCU_NAME "samd21e18"
+
+#define CIRCUITPY_MCU_FAMILY samd21
+
+#define MICROPY_HW_LED_STATUS   (&pin_PA14)
+
+#define SPI_FLASH_BAUDRATE  (8000000)
+
+// On-board flash
+#define SPI_FLASH_MOSI_PIN          &pin_PA16
+#define SPI_FLASH_MISO_PIN          &pin_PA18
+#define SPI_FLASH_SCK_PIN           &pin_PA17
+#define SPI_FLASH_CS_PIN            &pin_PA19
+
+// These are pins not to reset.
+#define MICROPY_PORT_A        (0)
+#define MICROPY_PORT_B        (0)
+#define MICROPY_PORT_C        (0)
+
+// If you change this, then make sure to update the linker scripts as well to
+// make sure you don't overwrite code.
+#define CIRCUITPY_INTERNAL_NVM_SIZE 256
+
+#define BOARD_FLASH_SIZE (0x00040000 - 0x2000 - CIRCUITPY_INTERNAL_NVM_SIZE)
```

Comment:
You can delete this; it's not used in atmel-samd now.

Reply:
Cleaned up board definitions per this conversation.